lib/ex_activecampaign/api_v3.ex

```diff
@@ -251,7 +251,10 @@ defmodule ExActivecampaign.ApiV3 do
         body
 
       x when x in 400..499 ->
-        %{error_message: "Bad Request", errors: body.errors}
+        case Map.has_key(body, :errors) do
+          true -> %{error_message: "Bad Request", errors: body.errors}
+          false -> %{error_message: "Bad Request", error: body.error}
+        end
```

Comment:
This is missing the `?` in `has_key?`

Could maybe make use of case's pattern matching here:
```suggestion
case body do
  %{errors: _} -> %{error_message: "Bad Request", errors: body.errors}
  %{error: _} -> %{error_message: "Bad Request", error: body.error}
end
```